docs/source/index.md

```diff
@@ -51,3 +51,10 @@ user-manual/index
 contributing/index
 release-notes/index
 ```
+
+````{ifconfig} context in ("volto")
```

Comment:
Is this not executed because `in ("volto")` evaluates to iterating over the characters in a string?

```suggestion
````{ifconfig} context in ("volto",)
```

Reply:
`context` evaluates to the string `volto`, so:

```pycon
>>> "volto" in ("volto")
True
>>> "volto" in ("volto",)
True
```

It makes no difference, although it was a good idea. I have been bitten by omitting that trailing comma many times in other code.